docs/GCP.md

```diff
@@ -0,0 +1,38 @@
+# Google Kubernetes Engine
+
+[![Gitter](https://img.shields.io/gitter/room/openwisp/general.svg)](https://gitter.im/openwisp/general)
+[![Support](https://img.shields.io/badge/support-orange.svg)](http://openwisp.org/support.html)
+
+**Right now, this is only tentative guide. Errata may exist. Please report errors on the [gitter channel](https://gitter.im/openwisp/general).**
+
+The following steps should yeild a working instance of openwisp:
+
+1. External IP: Create a global static IP and 2 IP address in the same region as you want to create your cluster in.
+2. Configure your domain with following A records:
+
+    ```
+    dashboard.openwisp.org      GLOBAL-IP
+    controller.openwisp.org     GLOBAL-IP
+    radius.openwisp.org         GLOBAL-IP
+    topology.openwisp.org       GLOBAL-IP
+    freeradius.openwisp.org     REGIONAL-IP-1
+    openvpn.openwisp.org        REGIONAL-IP-2
```

Comment:
Maybe we should use a place holder or variable like `dashboard.<your.domain>`.

Reply:
Yes, that sounds much better to avoid any confusion. :smile: 
On it. :+1: